### PR TITLE
Add baseapp-pages package and graphql improvements to baseapp-auth and baseapp-core

### PR DESCRIPTION
**Pull request by nossila**
### baseapp-pages

Adds this new package to handle flat pages

### baseapp-auth

Adds `is_staff`, `groups` and `user_permissions` to `AbstractUserAdmin`

### baseapp-core[graphql]
Adds `LanguagesEnum`, `DeleteNode` helpers and `JWTAuthentication` to handle JWT in our GraphQL endpoint

### CODE REVIEW

**Review by ficast — APPROVED** [review]

**Review by anicioalexandre — APPROVED** [review]
nice readme for the pages app!

File: baseapp-pages/baseapp_pages/models.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    )
+    meta_robots = models.CharField(max_length=100, null=True, blank=True)
+
+    meta_og_type = models.CharField(max_length=100, null=True, blank=True)
+    meta_og_image = models.ImageField(
+        upload_to=random_name_in("pages/metadata/og_image"), blank=True, null=True
+    )
+
+    class Meta:
+        unique_together = ("target_content_type", "target_object_id", "language")
```
Comment (nick-prat) [inline baseapp-pages/baseapp_pages/models.py:73]: Maybe we want to index on `target_content_type` & `target_object_id` like the URLPath model?

**Comment by nick-prat** [comment]
Just one comment, other than that it looks good to me!  I can't comment too much on the GraphQL part though, as I'm not too familiar with it yet.

File: baseapp-pages/baseapp_pages/models.py (hunk 1 above)
Comment (nossila) [reply baseapp-pages/baseapp_pages/models.py:73]: @nick-prat `unique_together` creates a btree index as well, no need to define it again

**Review by nick-prat — APPROVED** [review]

**Pull request merged**